ui/00_message/common/condition_name.xml

```diff
@@ -57,12 +57,12 @@
   <message>
     <key>CONDITION_NAME_21</key>
     <original>延焼</original>
-    <translation>Caught Fire</translation>
+    <translation>Burning</translation>
```

Comment:
Ok did it. The reason I changed it to Burning was to be consistent with the original Dragons Dogma which called the condition burning.